mrSerious/StackOverflow-lite
# #159909582 Update Application UI

#### What does this PR do?
Updates the UI as per LFA recommendation
#### Description of Task to be completed?
Add signup button to the navigation bar
Add upvote button
Add view for answers made by user
#### How should this be manually tested?
Visit the site [here](https://mrserious.github.io/StackOverflow-lite/) to see changes
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#159909582
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
